crates/fuel-core/src/state/historical_rocksdb/modifications_history.rs

```diff
@@ -39,3 +57,18 @@ where
         Column::HistoryColumn
     }
 }
+
+/// Blueprint for Modifications History V2. Keys are stored in big endian order
+/// using the `Column::HistoryColumnV2` column family.
+impl<Description> TableWithBlueprint for ModificationsHistoryV2<Description>
+where
+    Description: DatabaseDescription,
+{
+    type Blueprint = Plain<Primitive<8>, Postcard>;
+    type Column = Column<Description>;
+
+    // We store key-value pairs using the same column family as for V1
```

Comment:
I'm a bit confused here, since I see that the column is different, not the same.

Reply:
yes, stale comment. Removed in f162887b0c